### ISSUE TITLE
Does not update domoticz

### ISSUE THREAD

**Issue by TheOlfer** [stas-demydiuk/domoticz-zigbee2mqtt-plugin#516]
**Issue description**
After updating Zigbee2Mqtt to 1.16.2 and a plugin update via git pull, all devices won't update anymore. Some few were still able zu switch on/off, but also they won't update the status. Lastseen also remained at around the time the plugin was updated.

After reading, that upgrading from <3.0 may causes problems (well it did, read it too late) I removed the plugin from the Hardware, deleting all my Devices and then created it new. All Devices had been recreated with a new domoticzID. 
Still all Devices will not update at all. One Exception: api_transport

git pull will say: Already Up to date
Log: Found Errors with Plugin see below.

issue may relate to #496  

**Additional information**
Zigbee2MQTT version: 1.16.2
Python version: 3.7.3
Domoticz version: 2020.2 (12741)
Plugin version: 3.0

**Logs**
Domoticz Log showing Error:
```
2020-12-15 13:54:29.150 Error: (Zigbee2Mqtt) ----> Line 279 in '/home/pi/domoticz/plugins/zigbee2mqtt/plugin.py', function onMessage
2020-12-15 13:54:29.150 Error: (Zigbee2Mqtt) ----> Line 124 in '/home/pi/domoticz/plugins/zigbee2mqtt/plugin.py', function onMessage
2020-12-15 13:54:29.150 Error: (Zigbee2Mqtt) ----> Line 167 in '/home/pi/domoticz/plugins/zigbee2mqtt/mqtt.py', function onMessage
2020-12-15 13:54:29.150 Error: (Zigbee2Mqtt) ----> Line 153 in '/home/pi/domoticz/plugins/zigbee2mqtt/plugin.py', function onMQTTPublish
2020-12-15 13:54:29.150 Error: (Zigbee2Mqtt) ----> Line 24 in '/home/pi/domoticz/plugins/zigbee2mqtt/devices_manager.py', function set_devices
```

Full Domoticz Log after adding plugin as new Hardware
```
2020-12-15 13:54:28.484 Status: (Zigbee2Mqtt) Started.
2020-12-15 13:54:28.901 Status: (Zigbee2Mqtt) Entering work loop.
2020-12-15 13:54:28.902 Status: (Zigbee2Mqtt) Initialized version 3.0.0
2020-12-15 13:54:29.027 (Zigbee2Mqtt) MqttClient::subscribe
2020-12-15 13:54:29.078 (Zigbee2Mqtt) onMQTTSubscribed
2020-12-15 13:54:29.078 (Zigbee2Mqtt) Zigbee2mqtt bridge is online
2020-12-15 13:54:29.078 (Zigbee2Mqtt) MqttClient::publish zigbee2mqtt/bridge/config/groups ()
2020-12-15 13:54:29.079 (Zigbee2Mqtt) Zigbee2mqtt log level is info
2020-12-15 13:54:29.079 (Zigbee2Mqtt) Joining new devices is disabled on the zigbee bridge
2020-12-15 13:54:29.084 (Zigbee2Mqtt) Received available devices list from bridge
2020-12-15 13:54:29.084 (Zigbee2Mqtt) Coordinator address is 0x00124b0021600272
2020-12-15 13:54:29.084 (Zigbee2Mqtt) Creating domoticz device to handle "linkquality" key for device with ieeeAddr 0x00124b001b7d88d9
2020-12-15 13:54:29.085 (Zigbee2Mqtt) Creating domoticz device to handle "linkquality" key for device with ieeeAddr 0x00124b001f3205a5
2020-12-15 13:54:29.087 (Zigbee2Mqtt) Creating domoticz device to handle "linkquality" key for device with ieeeAddr 0x0017880106a652f2
2020-12-15 13:54:29.088 (Zigbee2Mqtt) Creating domoticz device to handle "battery_voltage" key for device with ieeeAddr 0x0017880106a652f2
2020-12-15 13:54:29.089 (Zigbee2Mqtt) Creating domoticz device to handle "battery" key for device with ieeeAddr 0x0017880106a652f2
2020-12-15 13:54:29.090 (Zigbee2Mqtt) Creating domoticz device to handle "action" key for device with ieeeAddr 0x0017880106a652f2
2020-12-15 13:54:29.091 (Zigbee2Mqtt) Creating domoticz device to handle "brightness" key for device with ieeeAddr 0x0017880106a652f2
2020-12-15 13:54:29.093 (Zigbee2Mqtt) Creating domoticz device to handle "linkquality" key for device with ieeeAddr 0x00158d0002ff699e
2020-12-15 13:54:29.094 (Zigbee2Mqtt) Creating domoticz device to handle "state_brightness_color" key for device with ieeeAddr 0x00158d0002ff699e
2020-12-15 13:54:29.095 (Zigbee2Mqtt) Creating domoticz device to handle "linkquality" key for device with ieeeAddr 0x00158d0003420102
2020-12-15 13:54:29.096 (Zigbee2Mqtt) Creating domoticz device to handle "state_brightness_color" key for device with ieeeAddr 0x00158d0003420102
2020-12-15 13:54:29.098 (Zigbee2Mqtt) Creating domoticz device to handle "linkquality" key for device with ieeeAddr 0xec1bbdfffe3157ce
2020-12-15 13:54:29.099 (Zigbee2Mqtt) Creating domoticz device to handle "state_brightness_color" key for device with ieeeAddr 0xec1bbdfffe3157ce
2020-12-15 13:54:29.100 (Zigbee2Mqtt) Creating domoticz device to handle "linkquality" key for device with ieeeAddr 0x00158d0002ff3f0f
2020-12-15 13:54:29.101 (Zigbee2Mqtt) Creating domoticz device to handle "state_brightness_color" key for device with ieeeAddr 0x00158d0002ff3f0f
2020-12-15 13:54:29.102 (Zigbee2Mqtt) Creating domoticz device to handle "linkquality" key for device with ieeeAddr 0xccccccfffeebc9d8
2020-12-15 13:54:29.104 (Zigbee2Mqtt) Creating domoticz device to handle "state_brightness_color" key for device with ieeeAddr 0xccccccfffeebc9d8
2020-12-15 13:54:29.105 (Zigbee2Mqtt) Creating domoticz device to handle "linkquality" key for device with ieeeAddr 0xccccccfffe89fa99
2020-12-15 13:54:29.106 (Zigbee2Mqtt) Creating domoticz device to handle "battery_voltage" key for device with ieeeAddr 0xccccccfffe89fa99
2020-12-15 13:54:29.107 (Zigbee2Mqtt) Creating domoticz device to handle "battery" key for device with ieeeAddr 0xccccccfffe89fa99
2020-12-15 13:54:29.108 (Zigbee2Mqtt) Creating domoticz device to handle "click" key for device with ieeeAddr 0xccccccfffe89fa99
2020-12-15 13:54:29.110 (Zigbee2Mqtt) Creating domoticz device to handle "brightness" key for device with ieeeAddr 0xccccccfffe89fa99
2020-12-15 13:54:29.111 (Zigbee2Mqtt) Creating domoticz device to handle "linkquality" key for device with ieeeAddr 0x00158d0003420b87
2020-12-15 13:54:29.112 (Zigbee2Mqtt) Creating domoticz device to handle "state_brightness_color" key for device with ieeeAddr 0x00158d0003420b87
2020-12-15 13:54:29.114 (Zigbee2Mqtt) Creating domoticz device to handle "linkquality" key for device with ieeeAddr 0x00178801081a4ade
2020-12-15 13:54:29.115 (Zigbee2Mqtt) Creating domoticz device to handle "value" key for device with ieeeAddr 0x00178801081a4ade
2020-12-15 13:54:29.116 (Zigbee2Mqtt) Creating domoticz device to handle "linkquality" key for device with ieeeAddr 0x0017880108a26a4e
2020-12-15 13:54:29.117 (Zigbee2Mqtt) Creating domoticz device to handle "value" key for device with ieeeAddr 0x0017880108a26a4e
2020-12-15 13:54:29.119 (Zigbee2Mqtt) Creating domoticz device to handle "linkquality" key for device with ieeeAddr 0x0017880104a735a5
2020-12-15 13:54:29.120 (Zigbee2Mqtt) Creating domoticz device to handle "value" key for device with ieeeAddr 0x0017880104a735a5
2020-12-15 13:54:29.121 (Zigbee2Mqtt) Creating domoticz device to handle "linkquality" key for device with ieeeAddr 0x00178801044093f4
2020-12-15 13:54:29.122 (Zigbee2Mqtt) Creating domoticz device to handle "value" key for device with ieeeAddr 0x00178801044093f4
2020-12-15 13:54:29.124 (Zigbee2Mqtt) Creating domoticz device to handle "linkquality" key for device with ieeeAddr 0x00158d00045cf83b
2020-12-15 13:54:29.125 (Zigbee2Mqtt) Creating domoticz device to handle "battery_voltage" key for device with ieeeAddr 0x00158d00045cf83b
2020-12-15 13:54:29.126 (Zigbee2Mqtt) Creating domoticz device to handle "battery" key for device with ieeeAddr 0x00158d00045cf83b
2020-12-15 13:54:29.127 (Zigbee2Mqtt) Creating domoticz device to handle "contact" key for device with ieeeAddr 0x00158d00045cf83b
2020-12-15 13:54:29.128 (Zigbee2Mqtt) Creating domoticz device to handle "linkquality" key for device with ieeeAddr 0x0017880100d09920
2020-12-15 13:54:29.130 (Zigbee2Mqtt) Creating domoticz device to handle "state_brightness_color" key for device with ieeeAddr 0x0017880100d09920
2020-12-15 13:54:29.131 (Zigbee2Mqtt) Creating domoticz device to handle "linkquality" key for device with ieeeAddr 0x00124b0003d067fe
2020-12-15 13:54:29.132 (Zigbee2Mqtt) Creating domoticz device to handle "state_brightness_color" key for device with ieeeAddr 0x00124b0003d067fe
2020-12-15 13:54:29.134 (Zigbee2Mqtt) Creating domoticz device to handle "linkquality" key for device with ieeeAddr 0x00158d0005404170
2020-12-15 13:54:29.135 (Zigbee2Mqtt) Creating domoticz device to handle "battery_voltage" key for device with ieeeAddr 0x00158d0005404170
2020-12-15 13:54:29.136 (Zigbee2Mqtt) Creating domoticz device to handle "battery" key for device with ieeeAddr 0x00158d0005404170
2020-12-15 13:54:29.137 (Zigbee2Mqtt) Creating domoticz device to handle "temperature" key for device with ieeeAddr 0x00158d0005404170
2020-12-15 13:54:29.139 (Zigbee2Mqtt) Creating domoticz device to handle "humidity" key for device with ieeeAddr 0x00158d0005404170
2020-12-15 13:54:29.140 (Zigbee2Mqtt) Creating domoticz device to handle "pressure" key for device with ieeeAddr 0x00158d0005404170
2020-12-15 13:54:29.141 (Zigbee2Mqtt) Creating domoticz device to handle "pressure" key for device with ieeeAddr 0x00158d0005404170
2020-12-15 13:54:29.143 (Zigbee2Mqtt) Creating domoticz device to handle "combined" key for device with ieeeAddr 0x00158d0005404170
2020-12-15 13:54:29.144 (Zigbee2Mqtt) Creating domoticz device to handle "combined" key for device with ieeeAddr 0x00158d0005404170
2020-12-15 13:54:29.147 (Zigbee2Mqtt) Creating domoticz device to handle "linkquality" key for device with ieeeAddr 0x0017880103dddbac
2020-12-15 13:54:29.148 (Zigbee2Mqtt) Creating domoticz device to handle "state_brightness_color" key for device with ieeeAddr 0x0017880103dddbac
2020-12-15 13:54:29.201 (Zigbee2Mqtt) Received groups list from bridge
2020-12-15 13:54:29.201 (Zigbee2Mqtt) Group Garten Terrassenlicht
2020-12-15 13:54:29.201 (Zigbee2Mqtt) Creating domoticz device to handle "state" key for device with ieeeAddr Garten Terra
2020-12-15 13:54:29.203 (Zigbee2Mqtt) Group Esstischlampe
2020-12-15 13:54:29.203 (Zigbee2Mqtt) Creating domoticz device to handle "state" key for device with ieeeAddr Esstischlamp
2020-12-15 13:54:29.204 (Zigbee2Mqtt) Group Loggialampe
2020-12-15 13:54:29.205 (Zigbee2Mqtt) Creating domoticz device to handle "state" key for device with ieeeAddr Loggialampe
2020-12-15 13:54:29.149 Error: (Zigbee2Mqtt) 'onMessage' failed 'TypeError':''NoneType' object is not subscriptable'.
2020-12-15 13:54:29.150 Error: (Zigbee2Mqtt) ----> Line 279 in '/home/pi/domoticz/plugins/zigbee2mqtt/plugin.py', function onMessage
2020-12-15 13:54:29.150 Error: (Zigbee2Mqtt) ----> Line 124 in '/home/pi/domoticz/plugins/zigbee2mqtt/plugin.py', function onMessage
2020-12-15 13:54:29.150 Error: (Zigbee2Mqtt) ----> Line 167 in '/home/pi/domoticz/plugins/zigbee2mqtt/mqtt.py', function onMessage
2020-12-15 13:54:29.150 Error: (Zigbee2Mqtt) ----> Line 153 in '/home/pi/domoticz/plugins/zigbee2mqtt/plugin.py', function onMQTTPublish
2020-12-15 13:54:29.150 Error: (Zigbee2Mqtt) ----> Line 24 in '/home/pi/domoticz/plugins/zigbee2mqtt/devices_manager.py', function set_devices
```

**Comment by faratou** [comment]
Same problem for me.

Additional information
Zigbee2MQTT version: 1.16.2
Python version: 3.7.3
Domoticz version: 2020.2 (12741)
Plugin version: 3.0

I have no error message, all devices won't update anymore in domoticz and none work. 

the devices update is ok in the tab "Custom" but nothing works in domoticz

**Comment by isom1266** [comment]
Same here
There are no errors after the update.
Go back to version 1.14.3 and everything works again

**Comment by Anjerlaan** [comment]
With zigbee2mqtt version 1.16.2 & plug-in v3.0
You need to add to the zigbee2mqtt configuration.yaml:

```
experimental:
  new_api: true
frontend:
  port: 1890 //choose a port which has no conflict with software that uses same port
```

Have you done this?

**Comment by Anjerlaan** [comment]
I did a clean install by the way....removed the hardware adapter, uninstall plug-in with plug-in manager.
Then the zigbee2mqtt plugin folder is removed.
After that follow the manual installation steps according to the latest read.me file

**Comment by TheOlfer** [comment]
Hi @Anjerlaan,

Thanks for replying!
I have the settings in my configuration.yaml like you mentioned, but with frontend port 8080. As it is the only application on that RPi I assume that's OK, also because Zigbee2mqtt is running flawlessly on that Pi. 
My Domoticz is on another Pi. I removed the adapter but I did not uninstall the plug in with the plug in manager. I just deleted the folder and made a clean new install.
The outcome was the same, unfortunately. Maybe there is something different with using the plug on manager. Never tried it before. 
I'll try this!

**Comment by TheOlfer** [comment]
Done... same errors again.

**Comment by faratou** [comment]
Hi @Anjerlaan,
Thanks you.

experimental:
  new_api: true

this line solved the problem of no integration in domoticz. Now, all is right.

**Comment by TheOlfer** [comment]
I've double checked this line and it is in the configuration. I restarted zigbee2mqtt. I saw that in domoticz the devices then had been updated. But only once at this this timestamp.

**Comment by Anjerlaan** [comment]
If you also add to configuration.yaml
 frontend: 8080 (choose different port as domoticz)

You can view a web based version of zigbee2mqtt (like domoticz)
But also change stuff like you normally send with payload to a zigbee2mqtt topic.

After change and restart zigbee2mqtt, Type in the web browser something like:
192.168.1.75:8080 (whatever ip adress your device uses)

**Comment by TheOlfer** [comment]
I did this too and I really like the Web frontend.. No success. But... 

Finally I've figured it out!!! @stas-demydiuk 

I have two devices, my Osram Garden Poles, that could not be discovered correctly from Zigbee2mqtt. For the Manufacturer and Model it states: Unsupported.
I have two of them since the beginning of using Zigbee2mqtt and Domoticz with no problems.

After deleting both from Zigbee2mqtt, it seems to work again! 

My System is quite a mess now... but this can be fixed.
Thanks for all for your hints! I really appreciate it!
